hadoop-ozone/dist/src/main/smoketest/ozonefs/hadoopo3fs.robot

```diff
@@ -36,4 +36,4 @@ Test hadoop dfs
     ${result} =        Execute                    hdfs dfs -ls ${dir}
                        Should contain             ${result}   ${PREFIX}-${random}
     ${result} =        Execute                    hdfs dfs -cat ${dir}/${PREFIX}-${random}
-                       Should contain             ${result}   This product includes software developed by The Apache Software
\ No newline at end of file
+                       Should contain             ${result}   This product includes software developed
```

Comment:
This is change intentional ?

Reply:
Also, has the output changed ? if so, whats the difference

Reply:
Yes, Hadoop's NOTICE.txt [has changed](https://github.com/apache/hadoop/commit/567091aa9b3f001208470345c673059c0a138b69#diff-b810ebe4ffda4293c79d292059b45b59de7e649e86314ffd5a1ee4eb4b1e08c8).  The relevant part:

```diff
-This product includes software developed by The Apache Software
-Foundation (http://www.apache.org/).
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
```

So I changed the test to only look for the common prefix.

Reply:
Ok thanks, makes sense to me. +1